--- a/drivers/usb/serial/option.c
+++ b/drivers/usb/serial/option.c
@@ -2073,6 +2093,14 @@ static int option_probe(struct usb_seria
 	if (iface_desc->bInterfaceClass == 0x08)
 		return -ENODEV;
 
+	/* Add for LongSung U9300 LTE modem */
+	if(serial->dev->descriptor.idVendor == cpu_to_le16(LONGSUNG_VENDOR_ID) &&
+		serial->dev->descriptor.idProduct == cpu_to_le16(LONGSUNG_U9300_PRODUCT_ID) &&
+ 		serial->interface->cur_altsetting->desc.bInterfaceNumber == 0){
+			printk(KERN_INFO"GZZ Discover the 0th interface for longsun U9300 ndis\n");
+			return -ENODEV;
+	}
+
 	/*
 	 * Don't bind reserved interfaces (like network ones) which often have
 	 * the same class/subclass/protocol as the serial interfaces.  Look at
