--- a/arch/arm/dts/Makefile
+++ b/arch/arm/dts/Makefile
@@ -140,12 +140,15 @@
 	rk3368-px5-evb.dtb \
 
 dtb-$(CONFIG_ROCKCHIP_RK3399) += \
+	rk3399-dlfr100.dtb \
 	rk3399-evb.dtb \
 	rk3399-eaidk-610.dtb \
 	rk3399-ficus.dtb \
 	rk3399-firefly.dtb \
 	rk3399-gru-bob.dtb \
 	rk3399-gru-kevin.dtb \
+	rk3399-guangmiao-g4c.dtb \
+	rk3399-h3399pc.dtb \
 	rk3399-khadas-edge.dtb \
 	rk3399-khadas-edge-captain.dtb \
 	rk3399-khadas-edge-v.dtb \
@@ -166,6 +169,7 @@
 	rk3399-rock-4se.dtb \
 	rk3399-rock-pi-4a.dtb \
 	rk3399-rock-pi-4c.dtb \
+	rk3399-mpc1903.dtb \
 	rk3399-rock960.dtb \
 	rk3399-rockpro64.dtb \
 	rk3399pro-rock-pi-n10.dtb
@@ -179,6 +183,10 @@
 	rk3566-soquartz-cm4.dtb \
 	rk3566-soquartz-model-a.dtb \
 	rk3568-bpi-r2-pro.dtb \
+	rk3568-r66s.dtb \
+	rk3568-opc-h68k.dtb \
+	rk3568-mrkaio-m68s.dtb \
+	rk3568-nanopi-r5s.dtb \
 	rk3568-evb.dtb \
 	rk3568-lubancat-2.dtb \
 	rk3568-nanopi-r5c.dtb \
